### PR TITLE
Implement color maps as static object

### PR DESCRIPTION
**Pull request by mgreter**
Makes the color tables available to all code parts!
Also saves the run-time initialization of the maps!
I actually used a perl script to generate the source file!

### CODE REVIEW

**Comment by mgreter** [comment]
@xzyfer this should be merged next ...

File: color_maps.cpp [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+  const char* color_to_name(int key)
+  {
+    auto p = colors_to_names.find(key);
+    if (p != colors_to_names.end()) {
+      return p->second.c_str();
+    }
+    return 0;
+  }
+
+  const char* color_to_name(Color* c)
```
Comment (xzyfer) [inline color_maps.cpp]: The header file defines this as `const char* color_to_name(const Color* c)`

**Comment by xzyfer** [comment]
Looks like a header definition mismatch. Otherwise :ship:

**Comment by mgreter** [comment]
Thanks, good catch! Finally this whole thing should be const, but for now this is not possible. It should be once the whole `sass_op` PR is merged!

**Comment by mgreter** [comment]
Wating for CI :traffic_light: to report back ...

**Pull request merged**